tests/common/test_configuration.py

```diff
@@ -32,8 +45,10 @@
           - bugs
           - daffy
           - tweety
-    """),
-    'file2': dals("""
+    """
```

Comment:
This was most likely darker auto-formatting or complaining about things. I remember have to make a bunch of weird updates to this file because it wasn't passing darker checks.

I'll give it another shot though.